Horan_Colby/Assignments/multiply.py

```diff
@@ -0,0 +1,6 @@
+def multiply(a, num):
+	for x in range(0, len(a)):
+		a[x] = a[x] * num
```

Comment:
good. in cases where you don't care about the indices, it's more efficient and simpler to iterate the elements directly, e.g.: `for ele in a: total+=ele`